Example/ViewController.swift

```diff
@@ -92,6 +92,8 @@ class ViewController: UIViewController {
         super.viewDidLoad()
         
         navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "settings"), style: .plain, target: self, action: #selector(openSettings))
+
+        navigationItem.rightBarButtonItem?.isEnabled = SettingsViewController.numberOfSettings > 0
```

Comment:
As I understood, this is done to temporary(?) disable settings. At the same time `numberOfSettings` is a let constant defined as zero. So, this condition will never be true. Maybe we should instead rely here on `SettingsViewController.settings().isEmpty` instead? Thus it will be automatically enabled when there are settings available.

Reply:
I thought about using `SettingsViewController.settings().isEmpty` and decided to add a property because the `settings` method in that case creates setting items which won't be used, and that's why in theory it can be too heavy just for getting the number of settings

Reply:
Good point. Maybe then rename it to `numberOfSections` to show it's relation to `sections()` below?

Reply:
renamed